community-members.md

```diff
@@ -64,6 +64,12 @@ Maintainers:
 
 ## Agent and Collector
 
+Repo: TBD
+
+Maintainers:
+
+- [Paulo Janotti](https://github.com/pjanotti)
```

Comment:
I am also very involved in this, so you can add me as well for this. Also we are going to use this inside Google (very soon) so it is good to have me as maintainer.